go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/talos-systems/os-runtime
 
 go 1.14
 
+replace gopkg.in/yaml.v3 => github.com/unix4ever/yaml v0.0.0-20210315173758-8fb30b8e5a5b
```

Comment:
I think it would be a good practice to add comments explaining all non-trivial go.mod directives: `replace`s, indirect dependencies, etc.

Reply:
good point, updated.